demos/embeddings/barbell-graphwave.ipynb

```diff
@@ -0,0 +1,162 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "m1 = 7\n",
```

Comment:
I just chose arbitrary numbers for the barbel graph, but I'll change things to match for the paper to verify things work.

Also, the scale sand sampling points are different which will affect the embeddings. I'll use the scales and sampling points they've got in the paper - that should fix things up.

Reply:
I've tried using the exact same parameters as the paper, with all the eigenvalues calculated exactly with numpy, and the scales calculated automatically but i still can't replicate this any better. 

This makes me think the difference here is the use of Chebyshev polynomials in the paper vs eigenvalues here. Which makes sense, the different approaches should yield different approximations

Reply:
Ok, could we add a note somewhere about this in the notebook, e.g.

```markdown
The plot here doesn't exactly match the one in the paper, which we think is because the details of approximating the wavelet diffusion differ (the paper uses Chebyshev polynomials, which isn't implemented in StellarGraph yet [#853](https://github.com/stellargraph/stellargraph/issues/853)).
```